lib/src/components/NxNavigationSidebar/NxNavigationSidebar.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+
+import NxButton from '../NxButton/NxButton';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, propTypes } from './types';
+
+import './NxNavigationSidebar.scss';
+
+const defaultLogo = require('../../assets/img/SON_hexagon_cropped.svg');
```

Comment:
Don't forget to replace this with one that includes product name text

Reply:
> Don't forget to replace this with one that includes product name text

Or should we just not have a default?  The hexagon was general enough that some downstream projects might use it.  But an image that includes product name text would not be general/reusable.

Reply:
> Don't forget to replace this with one that includes product name text

@dturner32 could you please provide us with a generic hexagon-and-text logo for Sonatype? To use as default for the sidebar.

Reply:
> Or should we just not have a default?

And make `logo` a mandatory prop in the sidebar? I'm fine with that if RSC is fine with that.

Reply:
> And make logo a mandatory prop in the sidebar? I'm fine with that if RSC is fine with that.

It makes sense to me